RELEASING.md

```diff
@@ -5,4 +5,4 @@ Release Checklist
 * Add the CHANGELOG.md entry for the release by looking at the PRs.
 * Commit `package.json` (for version number) and `CHANGELOG.md`.
 * Run the `./build.js` script to make sure the build runs successfully.
-* If it all looks good, run `./build.js --publish` which will compile the SDK, push it to NPM, and push a tag to the repo.
\ No newline at end of file
+* If it all looks good, run `./build.js --publish` which will compile the SDK, push it to NPM, and push a tag to the repo.
```

Comment:
My eyes aren't what they used to be, but I don't see any difference between these two lines.

Reply:
newline.